src/CoreShop/Component/Order/Model/Cart.php

```diff
@@ -226,4 +226,20 @@ public function setPaymentProvider($paymentProvider)
     {
         throw new ImplementedByPimcoreException(__CLASS__, __METHOD__);
     }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function getLocaleCode()
```

Comment:
we do have three different locale getters now: 

``` 
cart: getLocaleCode
order: getOrderLanguage
user: getLocale
```
mark them as deprecated and use consistent getters like `getLocale()` could be a good idea. But of course we also need to take care about a good migration process then.

Reply:
done here #403